compiler/ballerina-parser/src/test/resources/statements/block-stmt/block_stmt_assert_08.json

```diff
@@ -573,6 +573,292 @@
                   ]
                 }
               ]
+            },
```

Comment:
Shall we add a bal test maybe so that once disallowed with (https://github.com/ballerina-platform/ballerina-lang/issues/36187) we have a track of how the syntax changed

Reply:
We currently have several unit tests(both negative and positive) with `block-stmt`.
Please refer to the following.
https://github.com/ballerina-platform/ballerina-lang/blob/5ced1d7cb5422e67d450ddaa9d219fddcb658f50/tests/jballerina-unit-test/src/test/resources/test-src/statements/block/block-stmt.bal#L106-L142